skyportal/tests/conftest.py

```diff
@@ -8,15 +8,13 @@
 
 from baselayer.app import models
 from baselayer.app.config import load_config
-from baselayer.app.test_util import (
+
+from baselayer.app.test_util import (  # noqa: F401
```

Comment:
Are all of these imports actually required for the tests? If not we should remove those that aren't needed

Reply:
I think so. I removed all the imports that flake8 tagged as being unused, except the baselayer.app.test_util.driver as removing that broke the tests when I ran it.

Reply:
That's perfect, thanks